src/main/kotlin/io/snyk/plugin/ui/SnykSettingsDialog.kt

```diff
@@ -582,7 +647,7 @@ class SnykSettingsDialog(
         rootPanel.add(
             generalSettingsSpacer,
             panelGridConstraints(
-                row = 5,
+                row = 6,
```

Comment:
Same here :). What do these layout changes do? We didn't add anything to the root level, right?

Reply:
Fixed!